lib/graph3d/DataGroup.js

```diff
@@ -0,0 +1,240 @@
+var DataSet  = require('../DataSet');
+var DataView = require('../DataView');
+var Point3d  = require('./Point3d');
+var Range    = require('./Range');
+
+
+function DataGroup() {
```

Comment:
I totally agree that header commenting should be reviewed as well. However, I would really like to see this explicitly stated in the programming guidelines. It's something that developers should be aware of and should adhere to, throughout the entire codebase of `vis.js`.